### PR TITLE
chore: nix/win specific commands in test-harness

### PR DESCRIPTION
**Pull request by P0lip**
As suggested in https://github.com/stoplightio/spectral/pull/1171

**Checklist**

- [x] Tests added / updated
- [x] Docs added / updated

**Does this PR introduce a breaking change?**

- [ ] Yes
- [x] No

### CODE REVIEW

File: test-harness/index.ts [hunk 1]
```diff
@@ -17,6 +17,9 @@ describe('cli acceptance tests', () => {
   describe.each(files)('%s file', file => {
     const data = fs.readFileSync(path.join(cwd, file), { encoding: 'utf8' });
     const scenario = parseScenarioFile(data);
+
+    if (scenario.command === void 0) return;
```
Comment (nulltoken) [inline test-harness/index.ts:21]: @P0lip Will this mark the test as 'skipped' in the console? Wouldn't that be the case, how complex would it be to make it happen? That would be pretty handy to detect scenario that are not fully xplat...

Comment (P0lip) [reply test-harness/index.ts:21]: Nah, it won't. Can make it marked though.

**Comment by P0lip** [comment]
image
image

**Comment by nulltoken** [comment]
> Marked them as 'todo'

Perfect!

**Pull request merged**